src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -150,8 +160,9 @@ function describeID(id) {
 }
 
 var ReactComponentTreeHook = {
-  onSetChildren(id, nextChildIDs) {
+  onSetChildren(id: DebugID, nextChildIDs: Array<DebugID>): void {
     var item = get(id);
+    invariant(item, 'Item must have been set');
```

Comment:
I added two invariants to make sure that we ensure that the item has been properly extracted out. The fact that it is actually set depends on a ton of different files so it may be a good idea to enforce it at runtime, idk